src/handlers/localstorage/globalSettings.js

```diff
@@ -15,6 +16,11 @@ export const getKeychainIntegrityState = () =>
 export const saveKeychainIntegrityState = state =>
   saveGlobal(KEYCHAIN_INTEGRITY_STATE, state);
 
+export const getUserBackupState = () => getGlobal(USER_BACKUP_STATE, null);
```

Comment:
I think it'll be clearer if this function returned BackupStateTypes.immediate as the default value (and then can optionally update the BackupStateTypes immediate value to "immediate" instead of null